pkg/controller/perconaservermongodb/version.go

```diff
@@ -228,14 +229,22 @@ func (r *ReconcilePerconaServerMongoDB) ensureVersion(ctx context.Context, cr *a
 		return errors.Wrap(err, "failed to check if major update requested")
 	}
 
+	watchNs, err := k8s.GetWatchNamespace()
```

Comment:
please make sure that it will detect correctly that CW is used when we have 
```
        - name: WATCH_NAMESPACE
          value: ""
```

Reply:
yes, it works 

```
percona-version-service-558864dc75-jtkmz percona-version-service 1660729320     info    server request payload logged as grpc.request.content field{"system": "grpc", "span.kind": "server", "grpc.service": "version.VersionService", "grpc.method": "Apply", "grpc.request.content": {"msg": {"product":"psmdb-operator","operatorVersion":"1.13.0","apply":"disabled","databaseVersion":"5.0.7-6","kubeVersion":"v1.23.8-gke.1900","customResourceUid":"73030f0b-d2dc-480f-b057-feb75c4f0224","shardingEnabled":true,"clusterWideEnabled":true}}}
```